Packs/DevSecOps/TestPlaybooks/MinIO-test-playbook.yml

```
@@ -0,0 +1,440 @@
+id: playbook-MinIO-Test
+name: playbook-MinIO-Test
+version: -1
+fromversion: 4.5.0
```

```suggestion
fromversion: 5.0.0
```
The version should be consistent across all files in the Pack
(It's currently 5.0.0) Please fix this.